bindings/java/CMakeLists.txt

```diff
@@ -359,10 +359,9 @@ if(NOT OPEN_FOR_IDE)
     get_property(integration_jar_path TARGET fdb-integration PROPERTY JAR_FILE)
 
 
-    add_test(NAME java-integration 
-                    COMMAND ${CMAKE_COMMAND} 
-                    -E env "FDB_CLUSTER_FILE=${PROJECT_BINARY_DIR}/fdb.cluster"
-                    ${Java_JAVA_EXECUTABLE}
+    # add_fdbclient_test will set FDB_CLUSTER_FILE if it's not set already
+    add_fdbclient_test(NAME java-integration
```

Comment:
My only concern here is that it will only spawn a single database for all integration tests. From a performance perspective that's probably the best approach, but we'll need to be careful to make sure that every IT will operate over a different keyspace in order to avoid contamination. I think it's worth mentioning in the README so that at least new devs will have some warning.

Reply:
Done

Reply:
this will create a fdb cluster per test though. So I don't think interleaving keyspaces will be a problem

Reply:
oh wait, you're starting all test in one `ctest`. I see, in that case the above statement is correct.